codecov.yml

```diff
@@ -6,4 +6,8 @@ coverage:
         threshold: 5%
 codecov:
   require_ci_to_pass: no
-comment: no
+comment:
+    behavior: new
+    layout: "condensed_header, files, condensed_footer"
+    hide_project_coverage: TRUE
```

Comment:
why we should hide porject coverage here?

Reply:
Because on each PR this is too much information and will distract people. If every PR increases or keeps the percentage of coverage - that's a win, I guess.